x-pack/plugins/security_solution/public/common/components/search_bar/index.tsx

```diff
@@ -348,7 +357,6 @@ export const SearchBarComponent = memo<SiemSearchBarProps & PropsFromRedux>(
     prevProps.toStr === nextProps.toStr &&
     prevProps.updateSearch === nextProps.updateSearch &&
     prevProps.dataTestSubj === nextProps.dataTestSubj &&
-    deepEqual(prevProps.indexPattern, nextProps.indexPattern) &&
```

Comment:
Should the equivalent check on `sourcererDataView` be added here, or will the new `useMemo` above cause the appropriate updates?